src/document-creator/document-creator.mdx

```diff
@@ -177,3 +199,54 @@ The `"accept"` accepts an array of string, that are file types, which will set t
 If you want to accept all file types, remove the entire `"accept"` line from `"attachments"` object.
 
 ---
+
+## Document Storage field
+
+```json
+"documentStorage": {
+  "apiKey": "kNb15YYZ6N1zBlYd25cjj8PLgK6YAuvN9Gf7fPM1",
+  "url": "https://api-ropsten.tradetrust.io/storage"
+}
+```
+
+- Note: This document storage field is optional.
```

Comment:
shall we explain what the purpose of the document storage is?
i.e QR code 

and also maybe can add a section at the end can show the format and link to the actions url ADR

Reply:
ok